gptqmodel/models/base.py

```diff
@@ -412,30 +412,40 @@ def quantize(
             "retain_w": needs_lora,  # lora needs original w
         }
 
+        # rotate model
+        if self.quantize_config.rotation:
+            from gptqmodel.models.definitions.llama import LlamaGPTQ
+            from gptqmodel.models.definitions.qwen2 import Qwen2GPTQ
+            if not isinstance(self, (LlamaGPTQ, Qwen2GPTQ)):
+                raise ValueError(f"rotation only supports: llama/qwen2 model, "
+                                    f"current model is {self.__class__.__name__}")
+
+            if self.model.config.tie_word_embeddings:
+                print("Rotation requires word embeddings to be untied. Untying.")
```

Comment:
@smpanaro Can you replace with `print` with `log.info` here for consistency since all `prints` should be `logs` now.

Reply:
No problem, done!